src/content/test/semantics/table-headers.tsx

```diff
@@ -14,6 +14,11 @@ export const infoAndExamples = create(({ Markup }) => (
             headers, assistive technologies report relationships between cells that don't actually exist. Some users are likely to find the
             table's content confusing.
         </p>
+        <h3>From a user's perspective</h3>
+        <p>
+            <Link.HeadingsVideo>This short video on tables</Link.HeadingsVideo> shows how correct table markup helps people who use screen
```

Comment:
Is this intended to be `<Link.TablesVideo>` rather than `<Link.HeadingsVideo>`?

Reply:
yes :) fixed it